From a43d911d89dff0b751e830b1e051c7c4c3ea927d Mon Sep 17 00:00:00 2001
From: Peter Chen <b29397@freescale.com>
Date: Thu, 24 Jun 2010 15:47:42 +0800
Subject: [PATCH] ENGR00124629 mx5x: Add Watchdog config for mfg defconfig

If not, the board will be reset as no one
serices the watchdog.
At USB recovery mode, the watchdog will be
enabled at rom code.

Signed-off-by: Peter Chen <b29397@freescale.com>
---
 arch/arm/configs/imx5_updater_defconfig |   17 +++++++++++++++--
 1 files changed, 15 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx5_updater_defconfig b/arch/arm/configs/imx5_updater_defconfig
index ea6fd7e..801eca5 100644
--- a/arch/arm/configs/imx5_updater_defconfig
+++ b/arch/arm/configs/imx5_updater_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.31
-# Tue Jun  8 18:49:19 2010
+# Wed Jun 23 17:31:28 2010
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -525,6 +525,7 @@ CONFIG_SCSI_LOWLEVEL=y
 CONFIG_ATA=y
 # CONFIG_ATA_NONSTANDARD is not set
 # CONFIG_SATA_PMP is not set
+# CONFIG_SATA_AHCI_PLATFORM is not set
 CONFIG_ATA_SFF=y
 # CONFIG_SATA_MV is not set
 # CONFIG_PATA_PLATFORM is not set
@@ -826,7 +827,19 @@ CONFIG_HWMON=y
 CONFIG_SENSORS_ISL29003=y
 # CONFIG_THERMAL is not set
 # CONFIG_THERMAL_HWMON is not set
-# CONFIG_WATCHDOG is not set
+CONFIG_WATCHDOG=y
+CONFIG_WATCHDOG_NOWAYOUT=y
+
+#
+# Watchdog Device Drivers
+#
+# CONFIG_SOFT_WATCHDOG is not set
+CONFIG_MXC_WATCHDOG=y
+
+#
+# USB-based Watchdog Cards
+#
+# CONFIG_USBPCWATCHDOG is not set
 CONFIG_SSB_POSSIBLE=y
 
 #
-- 
1.5.4.4

